app/views/page/show.html.erb

```diff
@@ -208,7 +208,27 @@
             <% end %>
           </div>
 
-        <%# Text and Images %>
+          <%# Map Component %>
+        <% when 'PageMapComponent' %>
+          <%
+            description = component.description
+            title = component.title
+            next_component = @page.page_components.find_by(position: pc.position + 1)
+          %>
+          <% if title.present? %>
+            <div class="margin-bottom-2 font-sans-lg text-bold margin-top-9">
+              <%= content_tag(:span, title, class: 'margin-right-1') %>
+            </div>
+          <% end %>
+          <div class="margin-bottom-1 page-map-component">
+            <%= render partial: "page/page_map", locals: { component: component } %>
+          </div>
+          <% if description.present? %>
+            <div class="margin-top-2 grid-row <%= page_narrow_classes %> <%= next_component != nil && next_component.component_type == 'margin-bottom-9 ' %> font-sans-sm line-height-162<%= ' margin-bottom-9' if last_component === index %>">
+              <%= content_tag(:span, description, class: 'margin-right-1') %>
+            </div>
+          <% end %>
+          <%# Text and Images %>
```

Comment:
Can we move this back one indent, to look like:
```
<%# Text and Images %>
<% when 'PageCompoundBodyComponent' %>
```

The `<%# Text and Images %>` line should go back to where it was before this PR

Reply:
OK.. went through the file and made sure the indent is correct for the comment headers.